package.json

```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "start:logger": "LOGGER_ENABLED=true node node_modules/react-native/local-cli/cli.js start --reset-cache",
+    "start:android": "concurrently -r 'node node_modules/react-native/local-cli/cli.js start' './node_modules/.bin/react-native run-android'",
```

Comment:
Yes, I also thought about it. But I often saw that usually write  the commands with full paths (it seems, for greater compatibility)